### PR TITLE
Login/signup

### PR DESCRIPTION
**Pull request by HMTanzeel**
Bootstrap template on LogIn and SignUp page

Screenshot from 2022-02-09 15-34-22
Screenshot from 2022-02-09 15-34-31

### CODE REVIEW

File: app/views/sessions/new.html.erb [hunk 1]
```diff
@@ -1,25 +1,57 @@
...
+          </div>
 
-      <div class='col'>
-        <p><%= label_tag :password, nil, class: "para-class" %></p>
-        <%= password_field :password, nil, class: "inputmail" %>
-      </div>
+          <!-- Email input -->
+          <div class="form-outline mb-4">
+            <%= text_field :email, nil, class: "form-control form-control-lg",type:"email",
+              id:"form3Example3", placeholder:"Enter a valid email address"%>
```
Comment (aliammaar) [inline app/views/sessions/new.html.erb]: remove ID

Comment (aliammaar) [reply app/views/sessions/new.html.erb]: `type:"email"` missing space before `:`

File: app/views/sessions/new.html.erb [hunk 2]
```diff
@@ -1,25 +1,57 @@
...
+        </form>
       </div>
-    <div>
+    </div>
   </div>
-
+  <div class="d-flex flex-column flex-md-row text-center text-md-start justify-content-between py-4 px-4 px-xl-5 bg-primary">
+    <!-- Copyright -->
+    <div class="text-white mb-3 mb-md-0">
+      Copyright © 2020. All rights reserved.
```
Comment (aliammaar) [inline app/views/sessions/new.html.erb]: get the year dynamically using `<%= Time.now.year %>`

File: app/views/users/_form.html.erb [hunk 3]
```diff
@@ -10,41 +10,74 @@
...
-      <div class='col'>
-        <p><%= form.label :email, nil, class: "para-class" %></p>
-        <%= form.text_field :email, class: "inputmail" %>
-      </div>
+          
+          <div class="align-items-center">
+            <!-- Name input -->
+            <div class="col-md-7 form-outline mb-4">
+              <%= form.text_field :name, class: "form-control form-control-lg",type:"name",
+                id:"form4Example1", placeholder:"Enter Your Name"%>
```
Comment (aliammaar) [inline app/views/users/_form.html.erb]: remove `id:`

Comment (aliammaar) [reply app/views/users/_form.html.erb]: take care of proper spaces

File: Task management system [hunk 4]
```diff
@@ -0,0 +1,17 @@
+Task management system
```
Comment (aliammaar) [inline Task management system]: Why we committed this file?

File: app/views/sessions/new.html.erb [hunk 5]
```diff
@@ -1,25 +1,57 @@
- <% if alert %>
-          <p class='notice'><%= alert %></p>
- <% end %>
+<p class= 'notice'><%= alert %></p>
```
Comment (aliammaar) [inline app/views/sessions/new.html.erb]: it should be like 
`<p class='notice'><%= alert %></p>`

File: app/views/sessions/new.html.erb [hunk 6]
```diff
@@ -1,25 +1,57 @@
...
+<%# .....................................LOGIN FORM.................................. %>
 
-      <div class='col'>
-        <p><%= label_tag :password, nil, class: "para-class" %></p>
-        <%= password_field :password, nil, class: "inputmail" %>
+<section class="vh-100">
+  <div class="container-fluid h-custom">
+    <div class="row d-flex justify-content-center align-items-center h-100">
+      <div class="col-md-9 col-lg-6 col-xl-5">
+        <img src= "https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-login-form/draw2.webp" class="img-fluid"
```
Comment (aliammaar) [inline app/views/sessions/new.html.erb]: remove space after src=

File: app/views/sessions/new.html.erb [hunk 7]
```diff
@@ -1,25 +1,57 @@
...
+          alt= "Sample image">
```
Comment (aliammaar) [inline app/views/sessions/new.html.erb]: remove space after alt=

File: app/views/sessions/new.html.erb [hunk 8]
```diff
@@ -1,25 +1,57 @@
...
+        <form>
+          <div class="divider d-flex align-items-center my-4">
+            <p class="text-center fw-bold mx-3 mb-0">LOGIN</p>
+          </div>
+          <!-- Login form start  -->
+          <%= form_tag sessions_path, method: :post do |form| %>
+            <!-- Email input -->
+            <div class="form-outline mb-4">
+              <%= text_field_tag :email, nil, class: "form-control form-control-lg", type: "email",
+                placeholder: "Enter a valid email address" %>
```
Comment (aliammaar) [inline app/views/sessions/new.html.erb]: why this line break?

File: app/views/sessions/new.html.erb [hunk 9]
```diff
@@ -1,25 +1,57 @@
...
+                  Remember me
+                </label>
+              </div>
+              <a href="#!" class="text-body">Forgot password?</a>
+            </div>
+
+            <div class="text-center text-lg-start mt-4 pt-2">
+              <%= submit_tag 'Login', class: "btn btn-primary btn-lg signup" %>
+              <p class="small fw-bold mt-2 pt-1 mb-0">Don't have an account?
+              <%= link_to 'Register', new_user_path, class: "link-danger" %></p>
```
Comment (aliammaar) [inline app/views/sessions/new.html.erb]: indentation issue
it should be like 
<p>
  <%= link_to ...%>
</p>

File: app/views/users/_form.html.erb [hunk 10]
```diff
@@ -10,41 +10,74 @@
...
 
-      <div class='col'>
-        <p><%= form.label :mobile, nil, class: "para-class" %></p>
-        <%= form.number_field :mobile, class: "inputmail" %>
+<%# ............................temp......................... %>
+  <section class="vh-100">
+  <div class="container-fluid h-custom">
+    <div class="row d-flex justify-content-start align-items-center h-100">
+      <div class="col-md-9 col-lg-6 col-xl-5">
+        <img src= "https://mdbcdn.b-cdn.net/img/Photos/new-templates/bootstrap-login-form/draw2.webp" class="img-fluid"
```
Comment (aliammaar) [inline app/views/users/_form.html.erb]: remove extra space

File: app/views/users/_form.html.erb [hunk 11]
```diff
@@ -10,41 +10,74 @@
...
+          alt= "Sample image">
```
Comment (aliammaar) [inline app/views/users/_form.html.erb]: extra space

File: app/views/users/_form.html.erb [hunk 12]
```diff
@@ -10,41 +10,74 @@
...
 
-      <div class='col'>
-        <p><%= form.label :email, nil, class: "para-class" %></p>
-        <%= form.text_field :email, class: "inputmail" %>
-      </div>
+          
+          <div class="align-items-center">
+            <!-- Name input -->
+            <div class="col-md-7 form-outline mb-4">
+              <%= form.text_field :name, class: "form-control form-control-lg", type: "name",
```
Comment (aliammaar) [inline app/views/users/_form.html.erb]: fix unnecessary line break in views.

File: app/views/users/_form.html.erb [hunk 13]
```diff
@@ -10,41 +10,74 @@
...
+            <!-- Address input -->
+              <div class="col-md-7 form-outline mb-3">
+                <%= form.text_field :address, class: "form-control form-control-lg",
+                  type: "Address", placeholder: "Address" %>
+              </div>
+
+              <div class="text-center text-lg-start mt-4 pt-2">
+                <%= form.submit "SignUp", class: "btn btn-primary btn-lg signup" %>
+                <span class="small fw-bold mt-2 pt-1 mb-0">Have an account?
+                  <%= link_to 'Login', new_session_path, class: "link-danger" %></span>
```
Comment (aliammaar) [inline app/views/users/_form.html.erb]: end span at new line

File: app/views/users/_form.html.erb [hunk 14]
```diff
@@ -10,41 +10,74 @@
...
+            <!-- Password input -->
+            <div class="col-md-7 form-outline mb-3">
+              <%= form.password_field :password, class: "form-control form-c  ontrol-lg",
+                type: "password", placeholder: "Password" %>
+            </div>
 
-      <div>
-        <%= form.submit "SignUp", class: "btn" %>
+            <!-- Password Confirmation input -->
+            <div class=col-md-7 form-outline mb-3">
```
Comment (aliammaar) [inline app/views/users/_form.html.erb]: Starting qoutes?

File: app/views/sessions/new.html.erb [hunk 15]
```diff
@@ -1,25 +1,57 @@
...
 
-<h1>LogIn</h1>
-<%= form_tag sessions_path, method: :post do |form| %>
-  <div class='container'>
-    <div class='row flex-column'>
-      <div class='col'>
-        <p><%= label_tag :email, nil, class: "para-class"%></p>
-        <%= text_field_tag :email, nil, class: "inputmail" %>
-      </div>
+<%# .....................................LOGIN FORM.................................. %>
```
Comment (aliammaar) [inline app/views/sessions/new.html.erb]: why this?

File: app/assets/stylesheets/application.css [hunk 16]
```diff
@@ -140,3 +140,26 @@ button, .btn {
...
+@media (max-width: 450px) {
+  .h-custom {
+    height: 100%;
+  }
+}
+
+.signup {
+  padding-left: 2.5rem;
+  padding-right: 2.5rem;
+}
```
Comment (aliammaar) [inline app/assets/stylesheets/application.css]: add line break at the end of file

File: app/views/users/_form.html.erb (hunk 14 above)
Comment (HMTanzeel) [reply app/views/users/_form.html.erb]: Starting quotes of `form.submit` ?

**Review by aliammaar — APPROVED** [review]

**Pull request merged**